src/main/java/net/minecraftforge/common/ForgeModContainer.java

```diff
@@ -320,30 +320,22 @@ public void onConfigChanged(OnConfigChangedEvent event)
     {
         if (getMetadata().modId.equals(event.getModID()))
         {
-            if (!event.isWorldRunning())
+            if ("chunkLoader".equals(event.getConfigID()))
             {
```

Comment:
The point is that these configs do nothing while the server is running so why do we care to sync them when they would not take effect? If anything they should be moved to not be editable in the in-game menu. And if we're cleaning this up then we should move to the new Annotation based config system. I've just never got around to it.